RNA-seq/README.md

```diff
@@ -0,0 +1,12 @@
+# RNA-Seq Training Module
+
+This CCDL-designed module covers the analysis of RNA-seq data using Salmon.
+It is designed to be taught in approximately 3.5 hours.
+It depends on knowledge gained in the [intro to R](https://github.com/AlexsLemonade/training-modules/tree/master/intro-to-R-tidyverse) module and analyses are performed within a [Docker container](https://github.com/AlexsLemonade/training-modules/tree/master/docker-install).
+It covers QC steps, gene expression abundance estimation, and certain downstream analyses.
+The notebooks that comprise this module are:
+* [QC, trim, and quantification with Salmon](https://github.com/AlexsLemonade/training-modules/blob/master/RNA-seq/01-qc_trim_quant.md)
```

Comment:
Incredibly nit-picky, but can you pick a style for these bullet points (e.g., what gets capitalized) and apply it across them?  It's very possible you derived these bullet points from an inconsistent style in titling our notebooks but it's way more obvious here.

Reply:
Not sure what to do with tximport - what is our standard? Lower case because the lower case is part of the name (like refine.bio) or sentence case to Tximport?

Reply:
I would go with tximport

Reply:
I just lower cased things - do we like lower case or sentence case?

Reply:
I have a slight preference for sentence case 🤷‍♀

Reply:
sentence cased 👍